readme.md

```diff
@@ -11,34 +11,68 @@
 ## Install
 
 ```bash
-npm install --save react-fontawesome
+$ npm install --save react-fontawesome
 ```
 
-_Note: This component does not include any of the Font Awesome CSS or fonts, so you'll need to make sure to include those on your end somehow, either by adding them to your build process or linking to [CDN versions](https://www.bootstrapcdn.com/fontawesome/)._
```

Comment:
This was kept right next to install because people often reported not seeing the note above installing font-awesome. I would like to retain this information up here, even if it is duplicated below in some form.